node-dns-exporter/templates/alert-rules.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Add an if switch, check if prometheus monitoring is enabled and add an option (default true) to enable alertmanager metrics.